app/app/urls.py

```diff
@@ -487,7 +487,7 @@
     re_path(r'^results/?', retail.views.results, name='results'),
     re_path(r'^activity/?', retail.views.activity, name='activity'),
     re_path(r'^townsquare/?', townsquare.views.town_square, name='townsquare'),
-    re_path(r'^$', townsquare.views.landing_toggle, name='index'),
```

Comment:
can we remove `landing_toggle` itself cause i'm assuming it's dead code

Reply:
let me check what is that one for

Reply:
donee